lib/jwt-utils.js

```diff
@@ -364,6 +364,14 @@ function checkPayload(token, cb) {
       return cb(errors.INVALID_EXP());
     }
 
+    var clientExpired = iatParsed + config.expiration;
```

Comment:
I don't understand the name "clientExpired"... what about expLimit?

Also, what happens if there is no iat in the incoming token?